test/operator-insert.spec.ts

```diff
@@ -67,6 +67,22 @@ describe('insert operator unit tests', () => {
     expect(obj).to.eq(data[0]);
   });
 
+  it('it should insert using defaultValue if selection syntax fails', () => {
+    const data: any[] = [{ foo: 'foo', bar: [{ baz: 'baz' }] }];
+    const operator = new InsertOperator({ name: 'insert', select: 'bar[0].bazzz', defaultValue: 'default' });
```

Comment:
Maybe add a few tests for `defaultValue` of each accepted type (string, boolean, object, etc)?

Reply:
Done